src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -871,6 +871,62 @@ type TaskSeq =
     /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
     static member takeWhileInclusiveAsync: predicate: ('T -> #Task<bool>) -> source: TaskSeq<'T> -> TaskSeq<'T>
 
+    /// <summary>
+    /// Returns a task sequence that, when iterated, skips elements of the underlying sequence while the
+    /// given function <paramref name="predicate" /> returns <see cref="true" />, and then yields the remaining
```

Comment:
trues should be in `<c>` ?

Reply:
Yes, should be. And many other snippets like it. But F# IDE tooling does not respect `<c>`. In fact, it is extremely limited. So I cheat 😆. Best I can do until I get time to fix F# IDE rendering code.